CHANGELOG

```diff
@@ -1,3 +1,6 @@
+== 10.0.0 2017-03-06
+* Adds support for creating, retrieving, and polling Sources
```

Comment:
Changelog could use some fleshing out. There's been a lot of commits since the last release: https://github.com/stripe/stripe-ios/compare/v9.4.0...master